hoomd/PythonLocalDataAccess.h

```diff
@@ -296,6 +296,38 @@ template<class Output, class Data> class LocalDataAccess
         return Output::make(data, shape, strides, read_only);
         }
 
+    template<class T, class S, template<class> class U = GlobalArray>
+    Output getBufferExplicitSize(std::unique_ptr<ArrayHandle<T>>& handle,
```

Comment:
I'm not sure making this an overload of `getGlobalBuffer` makes sense, since the function is still for grabbing a local array, though the size of the array (the ragged neighborlist) cannot be inferred from local or global pdata information. Like mentioned elsewhere, this is a hack to keep using the `LocalDataAccess` machinery outside of what it was designed around. Maybe a refactor of how array lengths are inferred in `LocalDataAccess` is right here?

Reply:
That isn't what I was referring to. I was asking for the code to be moved below for now. @joaander and I talked and I will likely refactor the local access API since its uses are broadening. For now we will use your hack, and when this is merged, I will work on the change.

Reply:
Ohh sorry, now I see. That sounds good!